frontend/src/components/SequenceDisplayPreview.vue

```diff
@@ -123,18 +124,24 @@ const emits = defineEmits([
 
 const popupOpen = ref(false);
 
-const tabNames = [
-  'Schlüsselwörter',
-  'Löschen',
-  'Mit Lehrkräften teilen',
-  'Mit Teilnehmern teilen',
-];
+const tabNames = computed(() => {
+  return [
+    i18n.global.tc('sequence.tag', 2),
```

Comment:
Also here (see comment inside SequenceEditView) - Return `sequence.tag`, localize inside TabbedContainer?

Reply:
this would result in having the singular form ob tag (in german Schlüsselwort) as the tab

Reply:
so we would have to decide whether we want that

Reply:
Do we ever use the singular somewhere?

Otherwise, while I guess this isn't standard, we could change the key to `sequence.tags` and just always store the plural

Reply:
that would be true. we would also have change how the other tabs are "generated", although that is no big change